lib/omniauth-dsds/lib/user_builder.rb

```diff
@@ -0,0 +1,55 @@
+module Omniauth
+  module Dsds
+    class UserBuilder
+
+      class NullRawInfo < Hash
```

Comment:
Inheriting from Hash tends to lead to a load of surprises down the road, how do you feel about modifying this?